projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -733,6 +735,8 @@ public static function enqueue_block_editor_assets() {
 				'dismissedNotices'                => $publicize->get_dismissed_notices(),
 				'isInstagramConnectionSupported'  => $publicize->has_instagram_connection_feature(),
 				'isMastodonConnectionSupported'   => $publicize->has_mastodon_connection_feature(),
+				'isAutoConversionAvailable'       => $auto_conversion_settings->is_available( 'image' ),
+				'isAutoConversionEnabled'         => $auto_conversion_settings->is_enabled( 'auto-conversion' ),
```

Comment:
Same comment as in other places to use just one key, either `image` or `auto-conversion`